workflow-bot-app/src/test/java/com/symphony/bdk/workflow/engine/camunda/monitoring/converter/ActivityDomainConverterTest.java

```diff
@@ -0,0 +1,42 @@
+package com.symphony.bdk.workflow.engine.camunda.monitoring.converter;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+import com.symphony.bdk.workflow.monitoring.repository.domain.ActivityInstanceDomain;
+
+import org.camunda.bpm.engine.impl.persistence.entity.HistoricActivityInstanceEntity;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.util.Date;
+
+class ActivityDomainConverterTest {
+
+  @Test
+  void apply() {
+    ActivityDomainConverter converter = new ActivityDomainConverter();
+    Date start = new Date();
+    HistoricActivityInstanceEntity entity = new HistoricActivityInstanceEntity();
+    entity.setActivityId("activity");
+    entity.setActivityInstanceId("activity-instance-id");
+    entity.setActivityName("activity");
+    entity.setActivityType("scriptTask");
+    entity.setId("id");
+    entity.setProcessDefinitionKey("workflow");
+    entity.setProcessInstanceId("process-instance-id");
+    entity.setStartTime(start);
+    Date end = new Date();
+    entity.setEndTime(end);
+    Duration duration = Duration.between(start.toInstant(), end.toInstant());
+    entity.setDurationInMillis(duration.toMillis());
+    ActivityInstanceDomain domain = converter.apply(entity);
```

Comment:
For readabilitycan you please add an empty line to separate te entity creation, the method under test being called, and the asserts?